optuna/integration/skopt.py

```diff
@@ -323,15 +323,15 @@ def ask(self) -> Dict[str, Any]:
                 value = float(value)
             if isinstance(distribution, distributions.DiscreteUniformDistribution):
                 value = value * distribution.q + distribution.low
-            if isinstance(distribution, distributions.IntUniformDistribution):
```

Comment:
A distribution that is an instance of `IntUniformDistribution` will be `IntUniformDistribution` and `IntDistribution` so we have to unify the if-then rules.

Reply:
:+1:

Note - branching on old distributions will be removed in follow-ups.